From 403ad5dd8556721e5c6ffefd294a13e8bc5d03d5 Mon Sep 17 00:00:00 2001
From: Tobias Brunner <tobias@strongswan.org>
Date: Mon, 19 May 2014 14:53:24 +0200
Subject: [PATCH] pfkey: Always include stdint.h

On some systems (e.g. on Debian/kFreeBSD) that header is required when
including ipsec.h, on Linux we require it too when including pfkeyv2.h,
so to simplify things we just always include it.
---
 src/libhydra/plugins/kernel_pfkey/kernel_pfkey_ipsec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/libhydra/plugins/kernel_pfkey/kernel_pfkey_ipsec.c
+++ b/src/libhydra/plugins/kernel_pfkey/kernel_pfkey_ipsec.c
@@ -14,6 +14,7 @@
  * for more details.
  */
 
+#include <stdint.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 
@@ -24,7 +25,6 @@
 #ifdef HAVE_NET_PFKEYV2_H
 #include <net/pfkeyv2.h>
 #else
-#include <stdint.h>
 #include <linux/pfkeyv2.h>
 #endif
 
